kyuubi-server/src/main/scala/org/apache/kyuubi/engine/ProcBuilder.scala

```diff
@@ -142,7 +142,7 @@ trait ProcBuilder {
   }
 
   final lazy val processBuilder: ProcessBuilder = {
-    val pb = new ProcessBuilder(commands: _*)
+    val pb = new ProcessBuilder(commands.toStream.asJava)
```

Comment:
will this introduce the overhead?

Reply:
`toStream` uses a iterator, which prevents array copy to a list.

Reply:
it requires to be converted to Array eventually, so early transformation should be good too. would prefer to keep as-is

Reply:
OK, I got your point. Closing this PR.